src/attribute/Initializable.sol

```diff
@@ -41,10 +40,9 @@ abstract contract Initializable is IInitializable {
 
     /**
      * @notice Returns whether the contract is currently being constructed
-     * @dev {Address.isContract} returns false for contracts currently in the process of being constructed
      * @return Whether the contract is currently being constructed
      */
     function _constructing() private view returns (bool) {
-        return !Address.isContract(address(this));
+        return !(address(this).code.length > 0);
```

Comment:
Little nervous that they removed this, I think I read that EOF will break this check also.

How does their version of `Initializable` handle this initialization now?

Reply:
Oh nvm looks like they removed it for misuse reasons: https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3945.